File: test/fsharpTests/CharTest/toUppercaseTest.fs
testCase "toUppercase('a')" 
<| fun _ -> 
    let expected = 'A'
    Expect.equal expected (Char.toUppercase 'a') "error"
testCase "toUppercase('A')" 
<| fun _ -> 
    let expected = 'A'
    Expect.equal expected (Char.toUppercase 'A') "error"
testCase "toUppercase('7')" 
<| fun _ -> 
    let expected = '7'
    Expect.equal expected (Char.toUppercase '7') "error"
testCase "toUppercase('\233')" 
<| fun _ -> 
    let expected = '\233'
    Expect.equal expected (Char.toUppercase '\233') "error"

File: test/ocamlTests/CharTest/toUppercaseTest.ml
test "toUppercase('a')" (fun () -> expect (Char.toUppercase 'a') |> toEqual Eq.char 'A') ; 
test "toUppercase('A')" (fun () -> expect (Char.toUppercase 'A') |> toEqual Eq.char 'A') ; 
test "toUppercase('7')" (fun () -> expect (Char.toUppercase '7') |> toEqual Eq.char '7') ; 
test "toUppercase('\233')" (fun () -> expect (Char.toUppercase '\233') |> toEqual Eq.char '\233') ; 
